pkg/globalnet/controllers/ipam/ipam.go

```diff
@@ -469,6 +493,53 @@ func (i *Controller) handleRemovedPod(obj interface{}) {
 	}
 }
 
+func (i *Controller) handleRemovedSvcEx(obj interface{}) {
+	var key, name, namespace string
+	var err error
+
```

Comment:
Need to check for tombstone here.

Reply:
I did this way so that we don't even need to check for tombstone. We re-queue with the key and if service or annotation already cleaned up we stop requeuing.